.github/workflows/continuous-integration.yml

```
@@ -8,9 +8,87 @@
+            opcache.enable_cli=0
+
+      - name: Get composer cache directory
+        id: composer-cache
+        shell: bash
+        run: echo "dir=$(composer config cache-files-dir)" >> $GITHUB_OUTPUT
+
+      - name: Cache dependencies
+        uses: actions/cache@v4
+        with:
+          path: ${{ steps.composer-cache.outputs.dir }}
+          key: ${{ runner.os }}-php${{ matrix.php-version }}-composer-${{ hashFiles('composer.json', 'composer.lock') }}
```

actionlint (1.7.4)

property "php-version" is not defined in object type {dependencies: string; os: string; php: number}

(expression)